code/game/objects/items/teleportation.dm

```diff
@@ -375,7 +375,7 @@
 			current_location = previous
 			break
 
-	do_teleport(C, current_location, channel = TELEPORT_CHANNEL_BLINK)
+	do_teleport(C, current_location, channel = TELEPORT_CHANNEL_BLINK, ignore_check_teleport = TRUE) // We've already checked above
```

Comment:
If the check_teleport fails on the first turf then we will perform this teleport anyway despite check_teleport failing